bin/version-manager.js

```diff
@@ -28,6 +28,7 @@ cmd
   .option('--major', 'Only iterate on major versions of packages.')
   .option('--minor', 'Iterate over minor versions of packages (default).')
   .option('--patch', 'Iterate over every patch version of packages.')
+  .option('-a, --all', 'Installs all packages, not just ones that differ in version')
```

Comment:
I'd like to get feedback on this because by default this will only install packages that differ from previous run.  But when we wire up node 16 versioned tests we want this flag to be set to it always installs every package, but since it's a CLI option and we wrap this in a npm script it might be hard to do so

Reply:
We can set the CI to run the version tests with this flag on for Node 16 only when we get to that work. Looks like we would have to pipe in the flag to this [point](https://github.com/newrelic/node-newrelic/blob/main/bin/run-versioned-tests.sh#L24) through a package.json script adding a commandline option to the `run-versioned-tests.sh` script in our agent. Seems ok to me.

Reply:
its easy enough to pass options through the npm scripts we use. seems like a good start. if we find the perf hit isn't too bad (or not noticeable cause npm 7 is faster anyways) then we can prob flip the default.